src/main/java/com/epam/reportportal/spock/NodeInfoUtils.java

```diff
@@ -128,7 +128,7 @@ private static String unrollIterationDescription(IterationInfo iterationInfo, St
 		Object[] dataValues = iterationInfo.getDataValues();
 		if (!parameterNames.isEmpty() && dataValues != null) {
 			iterationDescription += "\n" + IntStream.range(0, parameterNames.size())
-					.mapToObj(i -> parameterNames.get(i) + ": " + dataValues[i].toString())
+					.mapToObj(i -> parameterNames.get(i) + ": " + (dataValues[i] == null ? "null" : dataValues[i].toString()))
```

Comment:
Just `String.valueOf(dataValues[i])` will be enough.

Reply:
Sure, Implemented this change and pushed the code.